src/languages/xml.js

```diff
@@ -41,7 +41,11 @@ function(hljs) {
         className: 'meta',
         begin: '<!DOCTYPE', end: '>',
         relevance: 10,
-        contains: [{begin: '\\[', end: '\\]'}]
+        contains: [
+          hljs.QUOTE_STRING_MODE,
+          hljs.APOS_STRING_MODE,
```

Comment:
What is the thinking with lines 45-46? Is this related to the entity fixes?

Reply:
I outsource this section to a separate request.